src/Agent.Worker/TaskCommandExtension.cs

```diff
@@ -559,7 +559,8 @@ public void Execute(IExecutionContext context, Command command)
         }
     }
 
-    public class ProcessTaskDebugCommand: IWorkerCommand
+    [CommandRestriction(AllowedInRestrictedMode=true)]
```

Comment:
this is just an example usage of this class. once this PR is approved, this attribute should be added to all the command classes that are allowed in restricted mode.

Reply:
I meant "example usage of this attribute on a class"